infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -93,6 +98,24 @@ fi
 git -C $LLVM_SRC checkout $LLVM_REVISION
 echo "Using LLVM revision: $LLVM_REVISION"
 
+if [ -n "$INTROSPECTOR_PATCHES" ]; then
+  # For fuzz introspector.
```

Comment:
Maybe this comment would be improved by explaining what this section does for introspector, since that part is hard to understand.